sympy/functions/special/delta_functions.py

```diff
@@ -99,16 +101,17 @@ def simplify(self, x):
         """
         from sympy.polys.polyroots import roots
 
-        if not self.args[0].has(x) or (len(self.args) > 1 and self.args[1] != 0 ):
+        wrt = hints.pop('wrt')
```

Comment:
I am using 
```py
try:
     wrt = hints.pop('wrt')
except KeyError:
     print ("KeyError: The \'wrt\' keyword is required for the DiracDelta hint")
```
But still in the error message it outputs like `KeyError: 'wrt'`  
I think that it should output as ` KeyError: The 'wrt' keyword is required for the DiracDelta hint `
Am I correct?